### PR TITLE
Update guide-des-citations-references-et-abreviations-juridiques.csl

### PR DESCRIPTION
**Pull request by rafjaf**
Abbreviate "traduit par" in "trad."

### CODE REVIEW

File: guide-des-citations-references-et-abreviations-juridiques.csl [hunk 1]
```diff
@@ -57,7 +57,7 @@
   </macro>
   <macro name="translator">
     <names variable="translator">
-      <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal" prefix=" traduit par ">
+      <name font-style="normal" prefix=" trad. " and="text" delimiter-precedes-last="never" sort-separator=" ">
```
Comment (bwiernik) [inline guide-des-citations-references-et-abreviations-juridiques.csl:60]:
```suggestion
      <name font-style="normal" and="text" delimiter-precedes-last="never" sort-separator=" ">
        <label form="verb-short"/>
```

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks! This should actually be shown by using the `label` element rather than using a prefix.

**Comment by rafjaf** [comment]
Thank you very much. I am sorry to ask, but what should I do next?
Rafaël

**Review by rafjaf — COMMENTED** [review]
I agree with the change. What should I do next?

**Comment by bwiernik** [comment]
I'll take care of it

**Comment by bwiernik** [comment]
Thanks! Sorry on the delay.

**Comment by rafjaf** [comment]
No problem, and thank you for your work!

**Pull request merged**